python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -986,24 +977,21 @@ class RandomForestClassifier(Base):
                             then please use the CPU based predict by \
                             setting predict_model = 'CPU'")
 
-        elif self.num_classes > 2:
-            raise NotImplementedError("Predict_proba for multi-class "
-                                      "classification models is currently not "
-                                      "implemented. Please check cuml issue "
-                                      "#1679 for more information.")
+        if num_classes != self.num_classes and num_classes != 'auto':
```

Comment:
I think we should remove the num_classes option from the `predict_proba` function as well.